arangod/VocBase/Methods/Collections.cpp

```diff
@@ -802,7 +798,7 @@ Collections::create(         // create collection
     auto res = validateCreationInfo(
         infos.front(), vocbase, allowEnterpriseCollectionsOnSingleServer,
         enforceReplicationFactor, isLocalCollection(infos.front()),
-        NameValidator::isSystemName(name), allowSystem);
+        false, allowSystem);
```

Comment:
The change from `NameValidator::isSystemName(name)` may be ok here, but I am not 100% clear.
Is this function only called for shards, not collections?

Reply:
The method is called "createShard" and takes a new ShardID as input => It can only be called for shards.

A Shard never matches the isSystemName pattern, hence i think this change is correct.